src/main/java/org/galatea/starter/domain/IexSymbol.java

```diff
@@ -10,7 +10,7 @@
 
   private String symbol;
   private String name;
-  private Date date;
+  private String date;
```

Comment:
Why did you change this object type to a string?

Reply:
See above